android-ktx/src/main/java/com/mirego/trikot/metaviews/MetaLabelBinder.kt

```diff
@@ -110,14 +82,35 @@ object MetaLabelBinder {
             textView.alpha = alpha
         }
 
-        textView.bindOnTap(metaLabel, lifecycleOwnerWrapper)
-
         metaLabel.backgroundColor.asLiveData()
             .observe(lifecycleOwnerWrapper.lifecycleOwner) { selector ->
-                selector.default?.toIntColor()?.let {
-                    textView.setBackgroundColor(it)
+                if (!selector.hasAnyValue) {
```

Comment:
I think you should use the `fun MetaSelector<Color>.toColorStateList()` helper in `ColorExtensions`